### DISCUSSION TITLE
how to convert jwt token to username

### DISCUSSION THREAD

**Discussion by marefati110** [strapi/strapi#6125] [Legacy Topics]
hello 
i want to get username who send request
in ```ctx.request``` just access header and token how can i convert it to username ??

**Comment by MattieBelt** [comment]
@marefati110 `ctx.state.user` should contain the user object (as used in [the example](https://strapi.io/documentation/3.0.0-beta.x/concepts/policies.html#how-to-create-a-policy) for creating an policy). You could use that in your controller.
